dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1307,7 +1309,7 @@ paths:
           description: default response
   /api/ga4gh/v2/extended/containers/{organization}:
     get:
-      operationId: entriesOrgGet
+      operationId: entriesOrgGet_1
```

Comment:
Not really part of the PR, but hunt down and set a (unique) operation id for this and the one below to stop them from bouncing back and forth.